core-kotlin-modules/core-kotlin-10/src/test/kotlin/com/baeldung/variableshadowing/VariableShadowingUnitTest.kt

```diff
@@ -0,0 +1,177 @@
+@file:Suppress("UNUSED_VARIABLE", "UNUSED_ANONYMOUS_PARAMETER")
+
+package com.baeldung.variableshadowing
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+
+class VariableShadowingUnitTest{
+
+    @Test
+    fun `test variable shadowing`(){
```

Comment:
Fix formatting - a few more throughout this class - maybe apply the auto formatter?
```suggestion
    fun `test variable shadowing`() {
```